pkg/collector/corechecks/snmp/internal/checkconfig/config_metadata.go

```diff
@@ -86,6 +86,74 @@ var LegacyMetadataConfig = MetadataConfig{
 	},
 }
 
+var TopologyMetadataConfig = MetadataConfig{
+	"lldp_remote": {
+		Fields: map[string]MetadataField{
+			"chassis_id_type": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.4",
+					Name: "lldpRemChassisIdSubtype",
+				},
+			},
+			"chassis_id": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.5",
+					Name: "lldpRemChassisId",
+				},
+			},
+			"interface_id_type": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.6",
+					Name: "lldpRemPortIdSubtype",
+				},
+			},
+			"interface_id": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.7",
+					Name: "lldpRemPortId",
+				},
+			},
+			"interface_desc": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.8",
+					Name: "lldpRemPortDesc",
+				},
+			},
+			"device_name": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.9",
+					Name: "lldpRemSysName",
+				},
+			},
+			"device_desc": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.4.1.1.10",
+					Name: "lldpRemSysDesc",
+				},
+			},
+			// TODO: Implement later lldpRemSysCapSupported and lldpRemSysCapEnabled
+			//   - 1.0.8802.1.1.2.1.4.1.1.11 lldpRemSysCapSupported
+			//   - 1.0.8802.1.1.2.1.4.1.1.12  lldpRemSysCapEnabled
+		},
+	},
+	"lldp_local": {
+		Fields: map[string]MetadataField{
+			"interface_id_type": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.3.7.1.2",
+					Name: "lldpLocPortIdSubtype",
+				},
+			},
+			"interface_id": {
+				Symbol: SymbolConfig{
+					OID:  "1.0.8802.1.1.2.1.3.7.1.3",
+					Name: "lldpLocPortID",
+				},
+			},
+		},
+	},
+}
```

Comment:
🥜  Since everything here seems to be `lldp` SNMP configs, would it make sense to move this to the `lldp` package?

Reply:
Can definitely wait